proposals/0029-cooperative-vector.md

```diff
@@ -749,64 +760,64 @@ explicitly checked for the combinations below.
 #### Usage Example
 
 ```c++
-// Check for CooperativeVector support and query properties for MatrixVectorMulAdd
-D3D12_FEATURE_DATA_D3D12_OPTIONSNN CoopVecSupport = {};
+// Check for matrix vector support and query properties for MatrixVectorMulAdd
+D3D12_FEATURE_DATA_D3D12_OPTIONSNN TierSupport = {};
 
-d3d12Device->CheckFeatureSupport(D3D12_FEATURE_D3D12_OPTIONSNN, &CoopVecSupport, 
+d3d12Device->CheckFeatureSupport(D3D12_FEATURE_D3D12_OPTIONSNN, &TierSupport, 
                                  sizeof(D3D12_FEATURE_DATA_D3D12_OPTIONSNN));
 
-if (CoopVecSupport.CooperativeVectorTier == D3D12_COOPERATIVE_VECTOR_TIER_1_0) {
+if (TierSupport.CooperativeVectorTier == D3D12_COOPERATIVE_VECTOR_TIER_1_0) {
     // PropCounts to be filled by driver implementation
     D3D12_FEATURE_DATA_COOPERATIVE_VECTOR CoopVecProperties = {0, NULL, 0, NULL, 0, NULL};
 
     // CheckFeatureSupport returns the number of input combinations for inference intrinsic
-    d3d12Device->CheckFeatureSupport(D3D12_FEATURE_COOPERATIVE_VECTOR, &CoopVecSupport, 
+    d3d12Device->CheckFeatureSupport(D3D12_FEATURE_COOPERATIVE_VECTOR, &CoopVecProperties, 
                                      sizeof(D3D12_FEATURE_COOPERATIVE_VECTOR));
 
     // Use MatrixVectorMulAddPropCount returned from the above
 
     // Use CheckFeatureSupport call to query only MatrixVectorMulAddProperties
-    UINT MatrixVectorMulAddPropCount = CoopVecSupport.MatrixVectorMulAddPropCount;
+    UINT MatrixVectorMulAddPropCount = CoopVecProperties.MatrixVectorMulAddPropCount;
     std::vector<D3D12_COOPERATIVE_VECTOR_PROPERTIES_INFERENCE> properties(MatrixVectorMulAddPropCount);
-    CoopVecSupport.pMatrixVectorMulAddProperties = properties.data();
+    CoopVecProperties.pMatrixVectorMulAddProperties = properties.data();
 
     // CheckFeatureSupport returns the supported input combinations for the inference intrinsic
-    d3d12Device->CheckFeatureSupport(D3D12_FEATURE_COOPERATIVE_VECTOR, &CoopVecSupport, 
+    d3d12Device->CheckFeatureSupport(D3D12_FEATURE_LINEAR_ALGEBRA_MATRIX_VECTOR, &CoopVecProperties, 
                                     sizeof(D3D12_FEATURE_DATA_COOPERATIVE_VECTOR));
                                                                 
     // Use MatrixVectorMulAdd shader with datatype and interpretation
     // combination matching one of those returned.
     
 } else {
-    // Don't use Cooperative Vector
+    // Don't use Cooperative Vector ops
 }
 ```
 
 ### Convert Matrix to desired layout and type
 
-The weight and bias matrices used in the cooperative vector intrinsics are
+The weight and bias matrices used in the Linear Algebra intrinsics are
```

Comment:
Is it right for me to call these Linear Algebra intrinsics?

Reply:
I think that has the right meaning there.